esx_service/vmdk_ops.py

```diff
@@ -1241,7 +1247,8 @@ def log_attached_volume(vmdk_path, kv_uuid, vol_name):
     '''
     Log appropriate message for volume thats already attached.
     '''
-    cur_vm = findVmByUuid(kv_uuid)
+    # in volume kv, uuid stored for a VM is vc uuid. So find VM by vc uuid.
+    cur_vm = findVmByUuid(kv_uuid, True)
```

Comment:
This would be incorrect for old volumes created and in-use before we added instanceUuid. Kv for these old volumes will continue to be biosUuid